x-pack/plugin/ccr/qa/chain/src/test/java/org/elasticsearch/xpack/ccr/ChainIT.java

```diff
@@ -71,4 +77,53 @@ public void testFollowIndex() throws Exception {
         }
     }
 
+    public void testAutoFollowPatterns() throws Exception {
```

Comment:
The purpose of this test appears to be to test auto-following against two clusters. I am fine with reusing the infrastructure from the chain tests for this but I have a few comments:
 - let us not use the `ChainIT` class as that one is about testing chain replication
 - let us rename the sub-project from `chain` since this is no longer about testing chain replication only

Reply:
Coming up with a good name for the `chain` qa module is difficult, since many qa modules names have `multi-cluster` in their name. 

What about removing the `multi-cluster` prefix from `multi-cluster-downgraded-to-basic-license`, `multi-cluster-with-incompatible-license`, `multi-cluster-with-non-compliant-license` and `multi-cluster-with-security` qa modules name. At the same time then we can merge `multi-cluster` and `chain` qa modules. The new qa module with the name `multi-cluster` will start 3 clusters, so tests from both modules should work. If you think this makes sense then I think we should do this in a followup PR.

Reply:
+1